src/leap/mail/incoming/service.py

```diff
@@ -729,7 +729,7 @@ def signal_deleted(doc_id):
             d.addCallback(signal_deleted)
             return d
 
-        d = self._inbox.add_raw_message(data, (self.RECENT_FLAG,))
+        d = self._inbox.add_msg(data, (self.RECENT_FLAG,))
```

Comment:
Ups, I see now that you have modified `add_msg`. I keep reading before complaining more.